mpes/fprocessing.py

```diff
@@ -2229,12 +2402,12 @@ def appendEAxis(self, E0, **kwds):
         if ('t0' in kwds) and ('d' in kwds):
             self.ecalib_t0 = kwds.pop('t0')
             self.ecalib_d = kwds.pop('d')
-            self.columnApply(mapping=b.tof2ev, rescolname='E', E0=E0, d=self.ecalib_d, t0=self.ecalib_t0, t=t, **kwds)
+            self.columnApply(mapping=b.tof2ev, rescolname="energy", E0=E0, d=self.ecalib_d, t0=self.ecalib_t0, t=t, **kwds)

```

Comment:
I changed this here just to avoid manually changing it later. As we had decided to call the energy axis 'energy' in the data at some point in the past. Also, the MoTe2 file and the test file have it stored as 'energy' which also shows up in the config. Could change it back to E and replace it with energy in the gather_metadata function in that case.